molgenis-swagger/src/main/resources/templates/view-swagger.ftl

```diff
@@ -22,6 +22,41 @@ securityDefinitions:
     in: header
     name: x-molgenis-token
 paths:
+  /scripts/{name}/start:
+    post:
+      tags:
+        - Scripts
+      summary: Starts a Script Job.
+      description: Will redirect the request to the jobs controller, showing the progress of the started ScriptJobExecution. The Script's output will be written to the log of the ScriptJobExecution. If the Script has an outputFile, the URL of that file will be written to the ScriptJobExecution's resultUrl.
+      parameters:
+        - name: name
+          in: path
+          description: The name of the script to start
+        - name: any
```

Comment:
Since the ```any``` field will never work, I think it's better to leave it out completely. And explain in the description that you can only run scripts without parameters.